### ISSUE TITLE
Got a 64bit build despite requesting VS16Win32

### ISSUE THREAD

**Issue by biojppm** [lukka/run-cmake#35]
Hi and thanks for your work.

I requested VS16Win32 for the generator, and yet [the result was a 64bit build](https://github.com/lemire/fast_float/runs/1403807091?check_suite_focus=true#step:3:21).

Is there something I'm missing, or is this an unexpected result?

For the record, the full log is this:
```shellsession
Run lukka/run-cmake@v2
  with:
    cmakeListsOrSettingsJson: CMakeListsTxtAdvanced
    cmakeListsTxtPath: D:\a\fast_float\fast_float/CMakeLists.txt
    buildDirectory: D:\a\fast_float\fast_float/../../_temp/windows
    cmakeBuildType: Release
    buildWithCMake: true
    cmakeGenerator: VS16Win32
    cmakeAppendedArgs: -DFASTFLOAT_TEST=ON
    buildWithCMakeArgs: --config Release
    cmakeSettingsJsonPath: D:\a\fast_float\fast_float//CMakeSettings.json
    useVcpkgToolchainFile: false
    configurationRegexFilter: .*
    useShell: true
tool: C:\Program Files\CMake\bin\cmake.exe
Generate project files with CMake
  -- Building for: Visual Studio 16 2019
  -- The CXX compiler identification is MSVC 19.27.29112.0
  -- Detecting CXX compiler ABI info
  -- Detecting CXX compiler ABI info - done
  -- Check for working CXX compiler: C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.27.29110/bin/Hostx64/x64/cl.exe - skipped
  -- Detecting CXX compile features
  -- Detecting CXX compile features - done
  -- No build type selected, default to Release
  -- Configuring done
  -- Generating done
  -- Build files have been written to: D:/a/_temp/windows
```

**Comment by lukka** [comment]
@biojppm using `run-cmake` in advanced mode ignores the [cmakeGenerator](https://github.com/lukka/run-cmake/blob/main/action.yml#L40) input.

**Comment by biojppm** [comment]
Can I suggest that a note be added to that line as a comment? I referred precisely to this line, and I saw the output on the log so I assumed all was ok. Also the mention to the requested generator in the log is a bit misleading. Another thing that would make it quicker to figure out is seeing the actual cmake command line.

To be fair, maybe the constraint is mentioned in the larger doc, but I admit I jumped straight to the file, so a note there would save us hurried types from a foot-shot. 

Anyway, thanks from clarifying.
